docs/content/reference/resources/memory.mdx

```diff
@@ -90,9 +107,10 @@ Request body includes session/query context and message array:
 
 ### Retrieve Messages
 
-**GET** `/messages?session_id={id}&query_id={id}`
+**GET** `/messages?session_id={id}&query_id={id}&limit={n}&offset={n}`
```

Comment:
can we explain how to use these ? what are the different cases ? offset=0, limit=0, default values if not set ?